### PR TITLE
fix: fix for footer in the report pages

### PR DESCRIPTION
**Pull request by GusBaamonde**
related to: https://github.com/FromDoppler/Doppler-UI-System/pull/156

fix-footer

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
     -->
 
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
-    <link rel="stylesheet" type="text/css" href="https://cdn.fromdoppler.com/doppler-ui-library/v3.9.5/css/styles.css">
+    <link rel="stylesheet" type="text/css" href="https://cdn.fromdoppler.com/doppler-ui-library/build472/css/styles.css">
```
Comment (fcoronelmakingsense) [inline public/index.html]: Remember change for the final version

Comment (cbernat) [reply public/index.html]: Good catch @fcoronelmakingsense

**Comment by andresmoschini** [comment]
❤️

**Pull request merged**